src/yadism/esf/exs.py

```diff
@@ -67,6 +67,10 @@ def xs_coeffs(kind, y, x=None, Q2=None, params=None):
         ym = 0
         yp = 1.0
         yL = y**2 / (2 * (y**2 / 2 + (1 - y) - (mn * x * y) ** 2 / Q2))
+    elif kind == "XSFPFCC":
+        INV_GEV_TO_PB = GEV_CM2_CONV / 100.0 # Pb
```

Comment:
I guess the comment should be rather `pb` (note the small p - as in pico) (just to not confuse with lead :upside_down_face: )

Reply:
To really nitpick `Pb` in this context would mean "PetaBarn", instead of "picoBarn", according to SI.

Reply:
This is definitely a fair point! In addition, in what we are doing we also need the observable name added to the `observable_names.yaml`. Let me restore this branch and add these changes here.

Reply:
Now in #163.